tests/js/common/shell/shell-index.js

```diff
@@ -29,7 +29,7 @@
 const jsunity = require("jsunity");
 const internal = require("internal");
 const errors = internal.errors;
-const testHelper = require("@arangodb/test-helper").Helper;
+const { Helper, versionHas } = require("@arangodb/test-helper");
```

Comment:
why `Helper` with a capital `H`?
```suggestion
const { helper, versionHas } = require("@arangodb/test-helper");
```

Reply:
oh, this will not work. As long as the function/object  comes from inside of testhelper, it must keep the name